template/scripts/generateDocs.js

```diff
@@ -65,7 +65,7 @@ function formatTemplateFileContents(content, destination) {
   result = result.replace(/>(\r\n|\r|\n){2,}/g, '>\r\n'); // Remove empty lines directly after a closing html tag.
   result = result.replace(/>(\r\n|\r|\n)```/g, '>\r\n\r\n```'); // Ensure an empty line before code samples.
   result = result.replace(/>(\r\n|\r|\n){2,}```(\r\n|\r|\n)/g, '>\r\n```\r\n'); // Ensure no empty lines before close of code sample.
-  result = result.replace(/([^(\r\n|\r|\n)])(\r\n|\r|\n)+#/g, "$1\r\n\r\n#"); // Ensure empty line before header sections.
+  // result = result.replace(/([^(\r\n|\r|\n)])(\r\n|\r|\n)+#/g, "$1\r\n\r\n#"); // Ensure empty line before header sections.
```

Comment:
Why was this removed? This code was there to make sure we had the necessary empty line before the header elements so that the MD docs rendered correctly.

If the concern is that the regex is inefficient than it needs to be replaced with a better expression, not removed.

Reply:
I tested this in different scenarios and I couldn't find a use case where this was addressing an issue? I am open to bringing this back if you can come up with a better regex, but since I couldn't repro the issue it's fixing I can't make it better.

Reply:
Test this in one of the components that use the latest version of this script. When I remove this line of code from the `generarteDocs.js` script in auro-datepicker and `npm run build:markdownDocs` I get the following changes in the .md file resulting in the following issue when MarkedJS rendering the content:
<img width="1487" alt="Screenshot 2023-05-05 at 2 18 44 PM" src="https://user-images.githubusercontent.com/23223137/236570972-0db2eb63-1630-4f2a-a650-dad705d59019.png">
<img width="1393" alt="Screenshot 2023-05-05 at 2 18 52 PM" src="https://user-images.githubusercontent.com/23223137/236570991-084feda3-157d-4e34-9d36-4b3bc92144f0.png">

Reply:
Issue removed from milestone and update reverted with this commit 

- https://github.com/AlaskaAirlines/WC-Generator/pull/383/commits/0c6ebae0b4722a560ff08596e556e032908caae7